### PR TITLE
Edited the procedure.

### PR DESCRIPTION
**Pull request by laidan6000**
### Type of change

_Select the type of your PR_

- Documentation

### Description

This pull request edits the procedure for deploying the topic operator using the cluster operator to make it clearer which file we are referring to.

Please can you check that I've formatted the cross-references correctly, thanks :)

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Write tests
- [ ] Make sure all tests pass
- [ y] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

File: documentation/book/proc-deploying-the-topic-operator-using-the-cluster-operator.adoc [hunk 1]
```diff
@@ -6,25 +6,40 @@
...
+----
+apiVersion: kafka.strimzi.io/v1alpha1
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+...
+  entityOperator:
+    topicOperator: {}
+    userOperator: {}
```
Comment (ppatierno) [inline documentation/book/proc-deploying-the-topic-operator-using-the-cluster-operator.adoc]: not sure if we should show the `userOperator` as well here, @tombentley  wdyt?

**Comment by laidan6000** [comment]
The build has failed.

**Comment by laidan6000** [comment]
Reformatted the cross-references in an attempt to resolve the asciidoctor build errors.

File: documentation/book/proc-deploying-the-topic-operator-using-the-cluster-operator.adoc (hunk 1 above)
Comment (laidan6000) [reply documentation/book/proc-deploying-the-topic-operator-using-the-cluster-operator.adoc]: I could highlight the topicOperator line in bold?

File: documentation/book/proc-deploying-the-topic-operator-using-the-cluster-operator.adoc [hunk 2]
```diff
@@ -6,25 +6,39 @@
 [id='deploying-the-topic-operator-using-the-cluster-operator-{context}']
 = Deploying the Topic Operator using the Cluster Operator
 
+This procedure describes how to deploy the Topic Operator using the Cluster Operator. Alternatively, you can deploy the Topic Operator as a standalone component. For more information, see xref:deploying-the-topic-operator-standalone[].
```
Comment (scholzj) [inline documentation/book/proc-deploying-the-topic-operator-using-the-cluster-operator.adoc]: I think we should clarify when should the alternative be used. The standalone deployment of the Topic Operator should be used when the user wants to use it with Kafka cluster not managed by Strimzi.

File: documentation/book/proc-deploying-the-topic-operator-using-the-cluster-operator.adoc [hunk 3]
```diff
@@ -6,25 +6,39 @@
...
 * A `Kafka` resource to be created or updated
 
 .Procedure
 
-. Topic Operator can be included in the Entity Operator.
-Edit the `Kafka` resource ensuring it has a `Kafka.spec.entityOperator` object that configures the Entity Operator.
-
+. Ensure that the `Kafka.spec.entityOperator` object exists in the `Kafka` resource. This configures the Entity Operator.
++
+[source]
```
Comment (scholzj) [inline documentation/book/proc-deploying-the-topic-operator-using-the-cluster-operator.adoc]: Add YAML as a format for this source block

File: documentation/book/proc-deploying-the-topic-operator-using-the-cluster-operator.adoc [hunk 4]
```diff
@@ -6,25 +6,39 @@
...
+----
+apiVersion: kafka.strimzi.io/v1alpha1
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+...
```
Comment (scholzj) [inline documentation/book/proc-deploying-the-topic-operator-using-the-cluster-operator.adoc]: Use comment to make this a valid YAML and use the correct indentation.

```suggestion
  #...
```

File: documentation/book/proc-deploying-the-topic-operator-using-the-cluster-operator.adoc (hunk 1 above)
Comment (laidan6000) [reply documentation/book/proc-deploying-the-topic-operator-using-the-cluster-operator.adoc]: It's suggested by the IBM Style Guide, so I'll give it a try.

File: documentation/book/proc-deploying-the-topic-operator-using-the-cluster-operator.adoc (hunk 3 above)
Comment (laidan6000) [reply documentation/book/proc-deploying-the-topic-operator-using-the-cluster-operator.adoc]: I don't think AsciiDoctor supports syntax highlighting for YAML. Given the time constraints, I suggest we merge this as-is and then investigate further.

File: documentation/book/proc-deploying-the-topic-operator-using-the-cluster-operator.adoc (hunk 4 above)
Comment (laidan6000) [reply documentation/book/proc-deploying-the-topic-operator-using-the-cluster-operator.adoc]: Done, thanks. :)

File: documentation/book/proc-deploying-the-topic-operator-using-the-cluster-operator.adoc (hunk 3 above)
Comment (laidan6000) [reply documentation/book/proc-deploying-the-topic-operator-using-the-cluster-operator.adoc]: OK, I see how this done.

**Pull request merged**